core/ipc/rpc/api/src/main/java/org/opennms/core/rpc/api/RpcClientFactory.java

```diff
@@ -35,7 +35,14 @@
  */
 public interface RpcClientFactory {
 
-    static final String LOG_PREFIX = "ipc";
+    // RPC Metrics related constants.
+    public static final String JMX_DOMAIN_RPC = "org.opennms.core.ipc.rpc";
+    public static final String LOG_PREFIX = "ipc";
```

Comment:
Should the LOG_PREFIX be lumped up with the other "RPC Metric related constants"?